### DISCUSSION TITLE
Add ability to save/load settings from file (or maybe even google drive)

### DISCUSSION THREAD

**Discussion by vscum** [qsniyg/maxurl#1139] [Ideas]
many other extensions have the option to save a config json file to download and import easily. The framework is already there to use a file input instead of copy and paste, just needs to be implemented. Also, im sure this would be more difficult but would be very nice to be able to use a cloud service like Gdrive to backup our settings. :)

**Comment by qsniyg** [comment]
Thanks for the suggestion!

Importing from a file is tracked at: https://github.com/qsniyg/maxurl/issues/543

I've tracked cloud storage at: https://github.com/qsniyg/maxurl/issues/1140. I'm not sure if I can implement this myself, because I don't have accounts for most cloud storage providers.

**Reply by vscum** [reply]
also maybe add a button in the quick extension menu to 'blacklist current site' from popups? thanks for workin on that storage one
